samples/sampler/stories/default/IncrementSlider.js

```diff
@@ -16,10 +16,8 @@ storiesOf('Agate', module)
 		'IncrementSlider',
 		() => (
 			<IncrementSlider
-				active={boolean('active', Config)}
```

Comment:
I'm not familiar with this component. Are the docs wrong?

active
"Sets the knob to selected state and allows it to move via 5-way"
focused
"Indicates focused state."

Should the docs be modified?

Reply:
The docs are described well. The `active` className is applied to `IncrementSlider` only when the enter key is pressed, it cannot be applied from knobs. I noticed that neither Sandstone has these two props as knobs, so I decided to remove them from Agate as well. Same issue appears for `Slider` as well.